src/state/airdrop/selectors.ts

```diff
@@ -1,10 +1,31 @@
 import { createSelector } from 'reselect';
 
+import { Wallet } from 'app/consts';
+import { HDSegwitP2SHAirWallet } from 'app/legacy';
 import { ApplicationState } from 'app/state';
 
+import { WalletsState } from '../wallets/reducer';
 import { AirdropState } from './reducer';
 
 const local = (state: ApplicationState): AirdropState => state.airdrop;
+const walletsState = (state: ApplicationState): WalletsState => state.wallets;
 
 export const thankYouSeen = createSelector(local, state => state.thankYouSeen);
 export const thankYouFlowCompleted = createSelector(local, state => state.thankYouFlowCompleted);
+export const airdropReadyWallets = createSelector(walletsState, state => {
+  return state.wallets.length > 0
+    ? state.wallets.filter((wallet: Wallet) => wallet.type === HDSegwitP2SHAirWallet.type)
+    : [];
+});
+
+export const subscribedWallets = createSelector([walletsState, local], (walletsState, airdropState) => {
```

Comment:
pass wallets selector here as the first argument from wallets/selector and subcribedIds selectors as a second argument

Reply:
did it bit different, as we dont have selector for subscribedIds (we dont use it in any view)

Reply:
done